### ISSUE TITLE
Cannot render type haxe.io.Encoding into a TypeScript declaration (TODO?)

### ISSUE THREAD

**Issue by sonygod** [nadako/hxtsdgen#37]
Cannot render type haxe.io.Encoding into a TypeScript declaration (TODO?)

`
  @:expose

   class Person {

    public var name:String;
    public function new(name:String) {
        
        this.name=name;
    }

    public function say(to:String) {
        
       var b= Bytes.alloc(10);
        trace('from haxe'+b);

    }
    
   }   `

**Comment by sonygod** [comment]
oh,this is only warning .instead of error.

**Comment by elsassph** [comment]
Yes not all types of abstract types can be converted to typescript. Maybe this one should be doable, hence the TODO.
